Buildings/Fluid/Humidifiers/EvaporativeCoolers/Baseclasses/IndirectWetCalculations.mo

```diff
@@ -0,0 +1,178 @@
+    "Primary air volume flow rate"
+    annotation (
+      Placement(
+      visible=true,
+      transformation(
+        origin={-120,-60},
+        extent={{-20,-20},{20,20}},
+        rotation=0),
+      iconTransformation(
+        origin={-140,-60},
+        extent={{-20,-20},{20,20}},
+        rotation=0)));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput VSec_flow(
+    final unit="m3/s",
+    displayUnit="m3/s",
+    final quantity = "VolumeFlowRate")
+    "Secondary air volume flow rate"
+    annotation (
+      Placement(
+      visible=true,
+      transformation(
+        origin={-120,-100},
+        extent={{-20,-20},{20,20}},
+        rotation=0),
+      iconTransformation(
+        origin={-140,-100},
+        extent={{-20,-20},{20,20}},
+        rotation=0)));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TDryBulPriIn(
+    final unit="K",
+    displayUnit="degC",
+    final quantity="ThermodynamicTemperature")
+    "Dry bulb temperature of the air at the inlet"
+    annotation (Placement(
+      visible=true,
+      transformation(
+        origin={-120,100},
+        extent={{-20,-20},{20,20}},
+        rotation=0),
+      iconTransformation(
+        origin={-140,100},
+        extent={{-20,-20},{20,20}},
+        rotation=0)));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TWetBulPriIn(
+    final unit="K",
+    displayUnit="degC",
+    final quantity="ThermodynamicTemperature")
+    "Wet bulb temperature of the inlet air"
+    annotation (Placement(
+      visible=true,
+      transformation(
+        origin={-120,60},
+        extent={{-20,-20},{20,20}},
+        rotation=0),
+      iconTransformation(
+        origin={-140,60},
+        extent={{-20,-20},{20,20}},
+        rotation=0)));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TDryBulSecIn(
+    final unit="K",
+    displayUnit="degC",
+    final quantity="ThermodynamicTemperature")
+    "Dry bulb temperature of the air at the inlet"
+    annotation (Placement(
+      visible=true,
+      transformation(
+        origin={-120,20},
+        extent={{-20,-20},{20,20}},
+        rotation=0),
+      iconTransformation(
+        origin={-140,20},
+        extent={{-20,-20},{20,20}},
+        rotation=0)));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TWetBulSecIn(
+    final unit="K",
+    displayUnit="degC",
+    final quantity="ThermodynamicTemperature")
+    "Wet bulb temperature of the inlet air"
+    annotation (Placement(
+      visible=true,
+      transformation(
+        origin={-120,-20},
+        extent={{-20,-20},{20,20}},
+        rotation=0),
+      iconTransformation(
+        origin={-140,-20},
+        extent={{-20,-20},{20,20}},
+        rotation=0)));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput TDryBulPriOut(
+    displayUnit="degC",
+    final unit="K",
+    final quantity="ThermodynamicTemperature")
+    "Dry bulb temperature of the outlet air"
+    annotation (Placement(
+     visible=true,
```

Comment:
There is no need to have the `visible=true`.

Reply:
Removed annotation flag